app/views/api/board_tickets/index.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.board_tickets do 
```

Comment:
this json structure doesn't feel right. it looks to be returning `board_tickets: [ <board_tickets> ... ]`

i think it should return `[<board_tickets> ... ]`

Reply:
I was copying how GitHub do it. So if you pull and issue you get
```
issue: { stuff: 1234 }
repo: { stuff_about_repo_the_issue_is_from: 5678 }
```
So you always get the result in a key of some sort

Reply:
really!!!??? well thats sucky... cleary someone hasn't been following the restful spec